src/Columns/ColumnUnique.h

```diff
@@ -122,6 +123,16 @@ class ColumnUnique final : public COWHelper<IColumnUnique, ColumnUnique<ColumnTy
         return false;
     }
 
+    double getRatioOfDefaultRows(double sample_ratio) const override
+    {
+        return getNestedColumn()->getRatioOfDefaultRows(sample_ratio);
```

Comment:
As long as all values are unique, and default value it 0th, it will always return 1 / size.
For CoulmnUnique this method could just throw, for LowCardinality we should call it for indexes